app/assets/stylesheets/responsive/_sidebar_style.scss

```diff
@@ -0,0 +1,45 @@
+/* Display of sidebars */
+.sidebar {
+  h2 {
+    font-size: 0.875em; //14px
+    text-transform: uppercase;
+    font-weight: 600;
+  }
+  small {
+    line-height: 1.5em;
+  }
+  .report-this-request {
+    text-align: center;
+  }
+}
+
+/* #follow_box .feed_link is here to override specificity in the request sidebar,
+   normally .feed_link should be enough */
+#follow_box .feed_link,
+.feed_link {
+  text-align: left;
+  .link_button_green {
+    text-align: center;
+  }
+}
+
+.request_short_listing {
+  h3 {
+    font-size: 1.125em;
+  }
+  p {
+    font-size: 0.875em;
+    line-height: 1.5em;
+  }
+}
+
+.act_link {
+  line-height: 1.5em;
+  a[href$="&via=whatdotheyknow"] {
```

Comment:
Or have a set of modifier classes?

```
.act_link--twitter {}
.act_link--wordpress {}
.act_link--write_to_them {} // I think this is UK-only
.act_link--widget {}
```

Either is fine by me – I mildly prefer the class-per-action.